src/styles/utils/_variable.scss

```diff
@@ -0,0 +1,19 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap");
+
+$ff: "Roboto", sans-serif;
+$fw-400: 400;
+$fw-500: 500;
+$fw-700: 700;
```

Comment:
to me it looks you overkilling it here 😄 
font-weight: 700 will be font-weight: 700 anyway

BUT the variable could be very useful it it were like `$heading-fw`, `$primary-fw` and so on

Reply:
I renamed them more flexable so it is not locked to the where I use but how I'd like to look.